hearth/spec/hearth/telemetry_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'opentelemetry-sdk'
+
+module Hearth
+  module Telemetry
+    describe '.otel_loaded?' do
+      before { Hearth::Telemetry.instance_variable_set(:@use_otel, nil) }
+
+      it 'is true when opentelemetry-sdk is available' do
+        expect(Hearth::Telemetry).to receive(:require)
+          .with('opentelemetry-sdk')
+          .and_return(true)
+        expect(Hearth::Telemetry.send(:otel_loaded?))
+          .to be_truthy
+      end
+
+      it 'returns false when opentelemetry-sdk is not available' do
+        expect(Hearth::Telemetry).to receive(:require)
+          .with('opentelemetry-sdk')
+          .and_raise(LoadError)
+        expect(Hearth::Telemetry.send(:otel_loaded?))
+          .to be_falsey
```

Comment:
For question mark methods, these should check true and false. not truthy and falsey